environment-dev.yml

```diff
@@ -36,8 +36,7 @@ dependencies:
   - psycopg2>=2.9.6
   - fastparquet>=2022.12.0
   - tqdm>=4.60.0
-  # pandas isn't compatible with numexpr=2.8.5: https://github.com/modin-project/modin/issues/6469
-  - numexpr<2.8.5
```

Comment:
2.8.5 still doesn't work in some cases, but we should take care of that since there are enough new versions working now. There is also a desire to pin versions like in pandas, if possible.

**Note:** 2.8.5 also doesn't always work in pandas.